src/test/e2e/api/openapi/openapi.e2e.test.ts

```diff
@@ -114,7 +114,7 @@ test('the generated OpenAPI spec is valid', async () => {
     } catch (err) {
         console.error(err);
         // there's an error here, so let's exit after showing it in the console.
-        expect(true).toBe(false);
+        expect(err).toBeUndefined();
```

Comment:
This gives more readable errors (we also have a console log above that gives even more details)